docs/quickstart.md

```diff
@@ -1,22 +1,102 @@
-## Quickstart
+# Quickstart
 
-- If you don't have [Node.js](https://nodejs.org/) installed, you may want to use [NVM](https://github.com/creationix/nvm).
-- Create API Description Document. In this tutorial, we'll use [API Blueprint](https://apiblueprint.org/) format and the `api-description.apib` filename.
+In following tutorial you can quickly learn how to test a simple HTTP API application with Dredd. The tested application will be very simple backend written in [Express.js][].
+
+### Prepare Your Environment
+
+If you don't have [Node.js](https://nodejs.org/) installed, you may want to use [nvm](https://github.com/creationix/nvm) and install it. Once `node` and `npm` commands exist on your computer, install Dredd globally:
 
 ```
-# GET /message
-+ Response 200 (text/plain)
+$ npm install -g dredd
+```
+
+## Document Your API
+
+First, let's design the API we are about to build and test. That means you will need to create an API description file, which will document how your API should look like. Dredd supports two formats of API description documents:
+
+- [API Blueprint][]
+- [Swagger][]
+
+If you choose API Blueprint, create a file called `api-description.apib` in the root of your project and save it with following content:
+
+```markdown
+FORMAT: 1A
 
-      Hello World!
+# GET /
++ Response 200 (application/json; charset=utf-8)
+
+        {"message": "Hello World!"}
 ```
 
-- Install Dredd.
+If you choose Swagger, create a file called `api-description.yml`:
 
+```yaml
+swagger: "2.0"
+info:
+  version: "1.0"
+  title: Example API
+  license:
+    name: MIT
+host: www.example.com
+basePath: /
+schemes:
+  - http
+paths:
+  /:
+    get:
+      produces:
+        - application/json; charset=utf-8
+      responses:
+        200:
+          description: ""
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+            required:
+              - message
 ```
-$ npm install -g dredd
+
+## Implement Your API
+
+As we mentioned in the beginning, we'll use [Express.js][] to implement the API. Install the framework by `npm`:
+
+```sh
+$ npm init
+$ npm install express --save
+```
+
+Now let's code the thing! Create a file called `app.js` with following contents:
+
+```javascript
+var app = require('express')();
+
+app.get('/', function(req, res) {
+  res.json({message: 'Hello World!'});
+})
+
+app.listen(3000);
 ```
 
-- Run interactive configuration:
+## Test Your API
+
+At this moment, the implementation is ready to be tested. Let's run the server as a background process and let's test it:
+
+```sh
+$ node app.js &
+```
+
+Finally, let Dredd validate whether your freshly implemented API complies with the description you have:
+
+```sh
+$ dredd api-description.apib http://localhost:3000  # API Blueprint
```

Comment:
I am not so familiar with Dredd params, so just ensure me, it won't start interactive config :-)